### PR TITLE
[V1] Entrypoints Test - Enable

### PR DESCRIPTION
**Pull request by robertgshaw2-redhat**
SUMMARY:
* enable tests

### CODE REVIEW

File: vllm/v1/structured_output/__init__.py [hunk 1]
```diff
@@ -40,7 +40,8 @@ def _delayed_init(self):
         tokenizer_group.ping()
 
         tokenizer = tokenizer_group.get_lora_tokenizer(None)
-        self.vocab_size = tokenizer.max_token_id + 1
+        self.mask_size = max(tokenizer.max_token_id,
+                             self.vllm_config.model_config.get_vocab_size())
```
[the remark this answers is not in the document]
Comment (russellb) [reply vllm/v1/structured_output/__init__.py:44]: but I'm definitely not confident in this ... I don't have a complete enough understanding of the factors involved here to know we're always getting the right value ...

[the remark this answers is not in the document]
Comment (robertgshaw2-redhat) [reply vllm/v1/structured_output/__init__.py:44]: Do we need the +1?

File: vllm/v1/structured_output/__init__.py [hunk 2]
```diff
@@ -85,7 +86,7 @@ def _delayed_init(self):
         self.executor = ThreadPoolExecutor(max_workers=max_workers)
         self._grammar_bitmask = xgr.allocate_token_bitmask(
             self.vllm_config.scheduler_config.max_num_seqs,
-            self.vocab_size,
+            self.mask_size,
```
Comment (robertgshaw2-redhat) [inline vllm/v1/structured_output/__init__.py:89]: Is the `bitmask` applied to the logits?

File: vllm/v1/structured_output/__init__.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (russellb) [reply vllm/v1/structured_output/__init__.py:44]: in the case that broke for me, get_vocab_size() returned the same as "max_token_id + 1".

There is another case, tested in CI, that needed the value "max_token_id" to work.

I'll probably run out of time today before I can get to the bottom of this properly...

File: vllm/v1/structured_output/__init__.py (hunk 2 above)
Comment (russellb) [reply vllm/v1/structured_output/__init__.py:89]: correct, applied here: https://github.com/vllm-project/vllm/blob/270a5da495d24e947a71e2fa0c56635f4fad2dc3/vllm/v1/worker/gpu_model_runner.py#L869-L914

**Comment by russellb** [comment]
The tests passed in CI, though the change is very suspicious. It could use more analysis and a proper explanation of the right thing, but it's definitely better than it was ...

I can try to dig into this for a proper explanation (and likely another change) to put this to bed for good next week

File: vllm/v1/structured_output/__init__.py (hunk 2 above)
Comment (robertgshaw2-redhat) [reply vllm/v1/structured_output/__init__.py:89]: It would make sense to me that this is the vocab_size then, since then it matches the shape of the logits

File: vllm/v1/structured_output/__init__.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (aarnphm) [reply vllm/v1/structured_output/__init__.py:44]: hmm, i think we should use `len(tokenizer.get_vocab())` to calculate it eagerly

https://github.com/vllm-project/vllm/pull/14851

**Comment by russellb** [comment]
Looks like @aarnphm ’s last fix for the xgrammar bitmask didn’t do the trick.

File: vllm/v1/structured_output/__init__.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (russellb) [reply vllm/v1/structured_output/__init__.py]: I rebased on that and it’s still failing

**Comment by russellb** [comment]
replaced by #14903
